### ISSUE TITLE
Bot does not accept the game

### ISSUE THREAD

**Issue by Deathronik** [Kyrluckechuck/TFT-Bot#59]
Bot does not accept the game, tried the ready release 0.9.11. Also tried to rescreen the game acceptance button itself, but still nothing works. All other functions in the bot work, except taking the game.

Log file:

[1673637550] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\TFT-Bot-main\captures/buttons/find_match.png', skipping
[1673637550] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\T-Bot-main\captures/buttons/find_match_highlighted.png', skipping
[1673637551] [MainThread] DEBUG || Clicking find match success: False
[1673637553] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\TFT-Bot-main\captures/buttons/find_match.png', skipping
[1673637554] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\FT-Bot-main\captures/buttons/find_match_highlighted.png', skipping
[1673637554] [MainThread] DEBUG || Clicking find match success: False
[1673637556] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\TFT-Bot-main\captures/buttons/find_match.png', skipping
[1673637557] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\FT-Bot-main\captures/buttons/find_match_highlighted.png', skipping
[1673637557] [MainThread] DEBUG || Clicking find match success: False
[1673637560] [MainThread] DEBUG || M|Could not find 'C:\Users\Deathronik\Desktop\TFT-Bot-main\captures/buttons/find_match.png', skipping
[1673637562] [MainThread] DEBUG || Clicking find match success: False

**Comment by Deathronik** [comment]
I would like to clarify that the bot starts searching for the game, but cannot accept

**Comment by Deathronik** [comment]
Perhaps the acceptance button has been changed, this is what it looks like at the moment
![accept](https://user-images.githubusercontent.com/46196966/212406380-133fd38e-a01c-433f-b31e-f1973155f430.png)

**Comment by Kyrluckechuck** [comment]
Dang, it looks like they updated that button as well -- I will try to get this updated this afternoon.

**Comment by Deathronik** [comment]
Anyway, I was able to solve the problem, the accept button is fine, the status inscription in the queue has been changed.

[1673637562] [MainThread] DEBUG || Clicking find match success: False

This exception was thrown because the code has a check that the player is in the queue, so the value took in False, and did not continue to accept the game.

By taking a new screenshot of the caption in the queue, everything worked. My screenshot:

![in_queue](https://user-images.githubusercontent.com/46196966/212419705-415f0383-f370-4aec-85e2-ae491f2e084f.png)
